pages/_document.tsx

```diff
@@ -43,6 +43,10 @@ class MyDocument extends Document {
           />
           <link rel="manifest" href="/manifest.json" />
           <link rel="shortcut icon" href="/favicon.ico" />
+          <link
+            href="https://fonts.googleapis.com/css2?family=Josefin+Slab&display=optional"
```

Comment:
based on https://nextjs.org/docs/basic-features/font-optimization, I dunno if we need something else for that